db/schema.rb

```diff
@@ -212,7 +212,7 @@
     t.integer  "client_data_id"
     t.string   "client_data_type", limit: 255
     t.integer  "requester_id"
-    t.string   "public_id"
+    t.string   "public_id",        limit: 255
```

Comment:
it keeps toggling back and forth because the production db schema has that constraint defined and I work off a copy of the production db.

I know @jessieay would be glad to see this noise removed. I will create a new PR to make that limit in the code.